torch/csrc/jit/codegen/cuda/test/test_gpu.cpp

```diff
@@ -7853,6 +7853,70 @@ TEST_F(NVFuserTest, FusionReductionScheduler_CUDA) {
       lparams);
 }
 
+TEST_F(NVFuserTest, FusionReductionWithTrivial_CUDA) {
```

Comment:
Can you please add a brief comment on what is checked by this test?

Reply:
added